### PR TITLE
MAP-22 Range Component

### PR DESCRIPTION
**Pull request by acazacu**
- Implemented the RangeInput component with unit test coverage
- Implemented an isDisabled flag throughout all inputs
- Fixed default values throughout all inputs

### CODE REVIEW

File: src/components/AppSidebar.vue [hunk 1]
```diff
@@ -6,12 +6,11 @@
 
 aside {
   background: white;
-  height: 100px;
+  height: 86px;
```
[the remark this answers is not in the document]
Comment (acazacu) [reply src/components/AppSidebar.vue]: They are temporary, but as everything is work in progress it's hard to remove them now. The end goal is is to have none or as few as possible fixed sizes defined in the codebase. I do my best to write the css like this, but getting it all out from the get go is almost impossible. These fixes are part of refactoring triggered by progressing work (this field alone refactors two other fields)

File: src/components/AppSidebar.vue [hunk 2]
```diff
@@ -6,12 +6,11 @@
...
   transition: height 0.2s ease-in-out;
 
   @media (min-width: $breakpoint-tablet-portrait) {
-    width: 244px;
-    padding: 24px 48px;
+    width: 340px;
     height: unset;
```
Comment (janjaap) [inline src/components/AppSidebar.vue]: Does the CSS postprocessor pick this up?? The `unset` value is not supported in the IEs: https://caniuse.com/#search=unset

File: src/components/input/RangeInput.vue [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+.range {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 48px;
+  background-color: white;
+  height: 76px;
+  cursor: pointer;
+  /*transition: height 0.5s ease-in-out, background-color 0.5s ease-in-out;*/
```
Comment (janjaap) [inline src/components/input/RangeInput.vue]: Forgot to remove?

File: src/components/input/RangeInput.vue [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  overflow: hidden;
+
+  > * {
+    padding: 0;
+  }
+
+  &.active {
+    flex-direction: column;
+    align-items: stretch;
+    justify-content: unset;
```
Comment (janjaap) [inline src/components/input/RangeInput.vue]: Same comment here about support for `unset`

File: src/components/AppSidebar.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (acazacu) [reply src/components/AppSidebar.vue]: That being said, it's the next PR that actually has me not needing these particular definitions. I'll remove them now, the rant above is useless

File: src/components/input/RangeInput.vue [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+    padding: 0;
+    cursor: default;
+    height: 230px;
```
[the remark this answers is not in the document]
Comment (acazacu) [reply src/components/input/RangeInput.vue]: removing

File: src/components/input/RangeInput.vue [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+@import "../../style/variables";
+
+.range {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 48px;
+  background-color: white;
+  height: 76px;
```
[the remark this answers is not in the document]
Comment (acazacu) [reply src/components/input/RangeInput.vue]: removing

File: src/components/input/RangeInput.vue (hunk 3 above)
Comment (acazacu) [reply src/components/input/RangeInput.vue]: ah yes, this was future work in progress :D

File: src/components/AppSidebar.vue (hunk 2 above)
Comment (acazacu) [reply src/components/AppSidebar.vue]: The CLI does have a post-processor that sweeps the code, I'm just not very educated on it, so I have no idea. I'm researching this now, good that you brought it up!

Comment (acazacu) [reply src/components/AppSidebar.vue]: Mkay, so I just checked, the post processor does NOT pick this up. I am looking into fixing this.

Comment (acazacu) [reply src/components/AppSidebar.vue]: I looked into this a bit, the best alternative ATM is to ditch usage of "unset".Or do you know of any postcss plugin that might solve this?

Comment (janjaap) [reply src/components/AppSidebar.vue]: No, not aware of any. Sorry.

Comment (acazacu) [reply src/components/AppSidebar.vue]: Alright, I removed the property all together and will refrain from using it. I was also using initial, turns out that it is also not supported by IE... Pity!

**Pull request merged**